functional-tests/src/test/groovy/specs/app/UploadDocumentSpec.groovy

```diff
@@ -0,0 +1,26 @@
+package specs.app
+
+import pages.app.AddEditProjectPage
+import pages.app.ProjectPage
+import pages.app.DocumentsPage
+
+class UploadDocumentSpec extends LoggedInSpec {
+
+  def projectName = "test project"
+  def convertedProjectName = "test-project"
+
+  def "Navigate to project page, upload project"() {
+    given: "I creat a project and navigate to it"
+      to AddEditProjectPage
+      page.projectName.value(projectName)
+      page.saveProject.click()// TODO: adjust this test to use a fixture for the porject - challenging because of authorization requirements
+
+      to ProjectPage, convertedProjectName, "detail"
+    when: "I click on the #ClickLink"
```

Comment:
#ClickLink only works if inside a test scenario that uses @Unroll. Please update the text for the *when* to be specific (there is no #ClickLink to be referenced). Same for #AssertPage.

Reply:
done